Go/1-simple/.gitignore

```diff
@@ -0,0 +1 @@
+1-simple
```

Comment:
What's the point of these `.gitignore` files?

Reply:
Ignoring builded binaries

Reply:
@DzyubSpirit, you can just use `go run` instead of building it every time, since these are just examples.